evap/staff/templates/staff_email_preview_form.html

```diff
@@ -12,8 +12,24 @@
                 <div class="form-field"><input class="form-control" name="email_subject{{ id_suffix }}" type="text" value="{{ email_template.subject }}" /></div>
             </div>
             <div class="form-group d-flex">
-                <div class="form-label">Email body</div>
-                <div class="form-field"><textarea class="form-control" cols="40" rows="10" name="email_body{{ id_suffix }}">{{ email_template.body }}</textarea></div>
+                <div class="form-label">
+                    <div class="nav nav-pills btn-switch btn-group-vertical float-right" role="tablist">
+                        <a class="p-1 space-normal font-weight-bold btn btn-light nav-link show active" href="#tab-plain{{ id_suffix }}" data-toggle="pill" role="tab" aria-controls="pills-plain{{ id_suffix }}" aria-selected="true">
```

Comment:
You're using an `id_suffix` for all ids here -- do we need them? For me, it looks like we could leave them out.

Reply:
Do you mean this `aria-controls="pills-plain{{ id_suffix }}"` for example? I think for accessibility the id_suffix would be useful here aswell but I dont know how much we focus on accessibility? I could also leave these aria things completely out if you want´.

Reply:
I'm talking about all locations where `{{ id_suffix }}` is used. Having aria-attributes is fine.

My point is that we could have `href="#tab-plain"` and `aria-controls="pills-plain"` -- we don't have multiple "items" here, so adding an id suffix doesn't help with anything, or am I missing something?

Reply:
In the case of an evalutation operation in which both contributors and participants can get reminded, this form is imported twice. And then we would have colliding `href` attributes which breaks the pill switching mechanism by bootstrap.
And for all aria things they are not needed, but I guess it would be consistent in the described case.

Reply:
That makes sense, thanks.